ext/civigrant/xml/Menu/Grant.xml

```diff
@@ -35,4 +35,18 @@
      <page_callback>CRM_Grant_Page_Tab</page_callback>
      <access_arguments>access CiviGrant</access_arguments>
   </item>
+  <item>
+     <path>civicrm/admin/options/grant_status/edit</path>
+     <title>Edit Grant Types</title>
+     <page_callback>CRM_Admin_Page_Options</page_callback>
+     <adminGroup>Option Lists</adminGroup>
+     <weight>385</weight>
+  </item>
+  <item>
+     <path>civicrm/admin/options/grant_type/edit</path>
+     <title>Edit Grant Status</title>
+     <page_callback>CRM_Admin_Page_Options</page_callback>
+     <adminGroup>Option Lists</adminGroup>
+     <weight>385</weight>
+  </item>
```

Comment:
These additions look good, but they also need to be removed from `Admin.xml`.

Reply:
Thanks @colemanw, I think I removed this paths from Admin.xml in the RelationshipType PR before it was merged.

Reply:
No it's still there in master:
https://github.com/civicrm/civicrm-core/blob/f0f09be1f7e59f4ad44a44fee5a67276fafb7678/CRM/Core/xml/Menu/Admin.xml#L590-L597

Reply:
ohh this wasn't added by the RelationshipType PR. I just took removed it in the latest commit.